frontend/src/bookings/buttons-calendar.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div>
```

Comment:
Are divs like this necessary? Do `template` tags require just a single child returned, like in React land?

Reply:
template tags can only contain one root element yes, and on that component <form></form> is the single root element so the <div> should not be necessary EXCEPT that the buttons component is rendered into a layout nested inside a flex-div container so it needs the top element to be a <div> in order to comply with the flex directive